### PR TITLE
[HAB-138] Images

### PR DESCRIPTION
**Pull request by AnkushSarkar10**
- Images moved out of firestore to firestorage
   - Images stored as `profilePic/<userid>.jpg`
   - moved picture out of the db types (we query storage to get it now)
- Moved to [Expo Image](https://docs.expo.dev/versions/latest/sdk/image/)
- [Image Picker](https://docs.expo.dev/versions/latest/sdk/image/)
- Image Resize with this [thing](https://docs.expo.dev/versions/latest/sdk/imagemanipulator/#saveformat)

### CODE REVIEW

**Comment by AnkushSarkar10** [comment]
dont merge

File: mobile-app/src/app/(app)/friends/editprofile.tsx [hunk 1]
```diff
@@ -22,7 +42,12 @@ export default function editprofile() {
     >
       {/* Form */}
       <View className="flex w-screen flex-col px-5">
-        {/* <ProfileCreationBoxes editPage={true} /> there was an error here; missing props */}
+        f
```
Comment (owengretzinger) [inline mobile-app/src/app/(app)/friends/editprofile.tsx]: remove this "f"

File: mobile-app/src/atoms/atoms.ts [hunk 2]
```diff
@@ -748,3 +725,32 @@ export const viewHabitDisplayTypeAtom = atomFamily((habitId: HabitIdT) =>
 export const settingAtom = atomFamily((settingKey: string) =>
   betterAtomWithStorage<number>(settingKey, 0),
 );
+
+// Images
+export const userPictureAtom = atomFamily(
+  ({ userId }: { userId: UserIdT }) =>
```
Comment (owengretzinger) [inline mobile-app/src/atoms/atoms.ts:731]: Why does this take an object as an argument? You could do it without

**Review by owengretzinger — CHANGES_REQUESTED** [review]
editprofile page pfp is not working for me

image

**Review by owengretzinger — APPROVED** [review]

**Pull request merged**